pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_uploader_tool.adoc

```diff
@@ -0,0 +1,66 @@
+[id="uploader_tool"]
+
+= Uploading content with the Uploader tool
+
+Many of the operations available in the Pantheon user interface can also be performed using commands in the command line interface.
```

Comment:
I suggest not referring to the Pantheon user interface in this module. I think "The pantheon uploader tool provides many Pantheon operations in the CLI" (something on these lines) should suffice.

Reply:
Fixed.